app/components/UpcomingEvents/index.js

```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes } from 'react';
+import UpcomingEvent from '../UpcomingEvent';
+
+const UpcomingEvents = ({ upcomingEvents }) => (
+  <section className="UpcomingEvents block">
+    <div className="content">
+      <h2>Upcoming Events</h2>
+    </div>
+    <div className="content space-between UpcomingEvents__events">
+      {upcomingEvents && upcomingEvents.map((event, index) => (
+        event ? <UpcomingEvent event={event} index={index} /> : null
```

Comment:
Why do you need to do 2 levels of error handling? How does a null value get into the arrayfield?

I think we should be able to remove the ternary operator.

Also apply the key at this level and then spread the `event` as props.